Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM hexpm/elixir:1.11.4-erlang-24.0-rc3-alpine-3.13.3 AS build
+
+RUN apk add --no-cache build-base git python3
```

Comment:
Why do we need python3? 🤔

Reply:
I assume it doesn't matter though, right? This is only for build.

Reply:
As of Alpine 3.12 there's no `python` package and you have to specify either `python2` or `python3` directly.

Reply:
> I assume it doesn't matter though, right? This is only for build.

Correct!